src/frontend/js/data/useStaticFilters/index.spec.tsx

```diff
@@ -68,24 +67,26 @@ describe('data/useStaticFilters', () => {
     expect(fetchMock.called('/api/v1.0/filter-definitions/')).toEqual(false);
 
     // useStaticFilters returns a promise for the static filter definitions
-    deferred.resolve(staticFilterDefinitions);
-    let filters;
     await act(async () => {
-      filters = await result.current();
+      deferred.resolve(staticFilterDefinitions);
+    });
+
+    await waitFor(async () => {
+      const filters = await result.current();
+      expect(filters).toEqual(staticFilterDefinitions);
     });
-    expect(filters).toEqual(staticFilterDefinitions);
+
     expect(fetchMock.calls('/api/v1.0/filter-definitions/').length).toEqual(1);
 
     fetchMock.restore();
     fetchMock.get('/api/v1.0/filter-definitions/', new Error('should not be called'));
-    // More calls return the filter but don't request on the API again
-    let filtersAgain;
-    await act(async () => {
-      filtersAgain = await result.current();
+    await waitFor(async () => {
+      const filtersAgain = await result.current();
+      expect(filtersAgain).toEqual(staticFilterDefinitions);
     });
-    expect(filtersAgain).toEqual(staticFilterDefinitions);
     expect(fetchMock.calls('/api/v1.0/filter-definitions/').length).toEqual(0);
   });
+  return;
```

Comment:
🤔

Reply:
uurrrggggghh forgot this one, and missed it during my pre-review.